lib/routes/dribbble/utils.js

```diff
@@ -2,11 +2,15 @@ const got = require('@/utils/got');
 const cheerio = require('cheerio');
 const url = require('url');
 
+// Refactored function to load a link asynchronously
 async function load(link) {
+    // Make a GET request to the specified  and retrieve the response
     const response = await got.get(link);
     const $ = cheerio.load(response.data);
 
-    const shotMedia = $('.media-shot,.main-shot') // join multiple shots together
+    // Join multiple shots together by selecting elements with class 'media-shot' or 'main-shot' or 'block-media-wrapper'
```

Comment:
You can use regex to parse the `shotData` data object in the HTML.
`/shotData:\s(\{.+\}),\n/` should work.